heron/api/src/java/com/twitter/heron/api/Config.java

```diff
@@ -147,6 +147,31 @@
    */
   public static final String TOPOLOGY_COMPONENT_RAMMAP = "topology.component.rammap";
   /**
+   * Is this topology stateful? The format of this flag is boolean
+   */
+  public static final String TOPOLOGY_STATEFUL = "topology.stateful";
```

Comment:
Would we need this globally, or per component? Assuming globally, could we give a more descriptive name like `topology.component.state.enabled` or something like that?